app/src/main/java/com/yagni/model/ProcessOne.java

```diff
@@ -67,68 +81,80 @@ public ProcessOne(){
 
         linkDb = new DbConnection(user, password);
     }
-	
-    public void updateInventory(){
-        // select the product and pull the quantity from the products table and assign it to a variable for that quantity
-        // Creating a statment object to pass a SQL statment to the database
+
+    public void updateInventory() {
+        // select the product and pull the quantity from the products table and assign
+        // it to a variable for that quantity
+        // Creating a statement object to pass a SQL statement to the database
         String totalStock, updatedStock;
         int intStock, intQuantity;
 
-        try{
-        Statement statement = linkDb.getConnection().createStatement();
-
-        // Calling the executeQuery method to execute a statement to the database
-        ResultSet resultSet = statement.executeQuery("SELECT * FROM yagni_inv_db.product WHERE product_id = '" + productId + "';");
-        resultSet.next();
-        totalStock = resultSet.getString(2);
-        
-        // Converting the variables to ints and updating the intStock by subtracting the input quantity from the value 
-        // gained from the select statement
-        intStock = Integer.parseInt(totalStock);
-        intQuantity = Integer.parseInt(quantity);
-        intStock = intStock - intQuantity;
-        updatedStock = String.valueOf(intStock);
-
-        //Use an update statement to update the products table in the database
-        statement.execute("UPDATE `yagni_inv_db`.`product` SET `quanity` = '" + updatedStock + "' WHERE (`product_id` = '" + productId +"');");
-
-        }catch (SQLException e){
+        try {
+            Statement statement = linkDb.getConnection().createStatement();
+
+            // Calling the executeQuery method to execute a statement to the database
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM yagni_inv_db.product" 
+                                                        + " WHERE product_id = '" + productId + "';");
```

Comment:
same issue here with the line-wrap. Just hit tab twice for the indent. Think you have like 6 tabs

Reply:
> same issue here with the line-wrap. Just hit tab twice for the indent. Think you have like 6 tabs

can keep this the same too, ignore comment